client/resources/aria-at-test-run.mjs

```diff
@@ -139,7 +139,7 @@ export function instructionDocument(resultState, hooks) {
             },
             `.`,
           ],
-          `Activate the "Open test page" button below, which opens the example to test in a new window${setupScriptDescription || '.'}`,
+          `Activate the "Open test page" button below, which opens the example to test in a new window${setupScriptDescription}`,
```

Comment:
May need to verify if the issue where the string `"undefined"` was being injected into instructions text for tests without setup scripts, is now gone.

Reply:
I still see it! I've deployed this branch to the sandbox, and if you sign into the sandbox and then go to this page you should see it: https://aria-at-app-sandbox.bocoup.com/run/120?user=5

Reply:
After clearing my cache I was able to verify that the issue is fixed! Since I've deployed this branch to the Sandbox, you can see it here (must be logged-in to the sandbox for the link to work) https://aria-at-app-sandbox.bocoup.com/run/120?user=5

Reply:
Actually, this is still occurring.

Reply:
This is now genuinely fixed (by https://github.com/w3c/aria-at/pull/576)